### PR TITLE
Add speedtest dashboard

### PR DESCRIPTION
**Pull request by mgarabed**
### What does this PR do?

Add default dashboard for speedtest integration.

(Updated) Screenshot: 
Screen Shot 2020-08-13 at 1 22 37 PM

### CODE REVIEW

File: speedtest/assets/dashboards/speedtest.json [hunk 1]
```diff
@@ -0,0 +1,325 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Speedtest",
+    "description": "",
```
Comment (ChristineTChen) [inline speedtest/assets/dashboards/speedtest.json:6]: Should there be a description with links to documentation?

File: speedtest/assets/dashboards/speedtest.json [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Ping Latency (ms)",
```
Comment (ChristineTChen) [inline speedtest/assets/dashboards/speedtest.json:87]: Should the titles follow our guidelines (note widget, gray background, 18px, pointer down)? 

https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices

File: speedtest/assets/dashboards/speedtest.json [hunk 3]
```diff
@@ -0,0 +1,325 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Speedtest",
```
Comment (ChristineTChen) [inline speedtest/assets/dashboards/speedtest.json:5]: Preset dashboard missing logo

File: speedtest/assets/dashboards/speedtest.json (hunk 2 above)
Comment (mgarabed) [reply speedtest/assets/dashboards/speedtest.json:89]: Good comment - updated accordingly.

**Pull request merged**